data/_ui/tooltips.txt

```diff
@@ -1275,6 +1275,9 @@ tip "Extended jump effects"
 tip "Ship outlines in shops"
 	`Controls the display of your fleet ship icons when in the shipyard or outfitter. When fancy, applies a sobel filter to all ships. (This can result in high GPU load for large fleets.) When fast, just uses the ship's sprite.`
 
+tip "Ship outlines in HUD"
+	`Controls the display of your flagship and target ship icons in the HUD. When fancy, applies a sobel filter to the sprites. (This can result in high GPU load for some ships on very old or low performance computers.) When fast, just uses the ships' sprites.`
```

Comment:
```suggestion
	`Controls the display of your flagship and target ship icons in the HUD. When fancy, applies a sobel filter to the sprites. (This can result in high GPU load for some ships on very old or low performance computers.) When fast, just uses the ship's sprite.`
```